docs/guides/qedma-qesem.ipynb

```diff
@@ -229,25 +229,35 @@
     "|`max_execution_time` | 0 < integer < 28,800 (8 hours)| Allows you to limit the QPU time, specified in seconds, to be used for the entire QESEM process. Please find additional details below.| 3,600 (one hour)|\n",
     "| `transpilation_level` | 0 / 1 | See description below | 1|\n",
+    "| `execution_mode` | `\"session\"` / `\"batch\"` | See description below | \"batch\"|\n",
+    "\n",
     "\n",
     "<Admonition type=\"caution\">The QPU time estimation changes from one backend to another. Therefore, when executing the QESEM function, make sure to run it on the same backend that was selected when obtaining the QPU time estimation. </Admonition>\n",
     "\n",
     "<Admonition type=\"note\">QESEM will end its run when it reaches the target precision or when it reaches `max_execution_time`, whichever comes first. </Admonition>\n",
     "\n",
-    "- `max_execution_time`: Allows you to limit the QPU time, specified in seconds, to be used for the entire QESEM process. Since the final QPU time required to reach the target accuracy is determined dynamically during the QESEM job, this parameter enables you to limit the cost of the experiment. If the dynamically-determined QPU time is shorter than the time allocated by the user, this parameter will not affect the experiment. The `max_execution_time` parameter is particularly useful in cases where the analytical time estimate provided by QESEM before the job starts is too pessimistic and the user wants to initiate a mitigation job anyway. After the time limit it reached, QESEM stops sending new circuits. Circuits that have already been sent continue running (so the total time may surpass the limit by up to 30 minutes), and the user receives the processed results from the circuits that ran up to that point. If you want to apply a QPU time limit shorter than the analytical time estimate, consult with Qedma to obtain an estimate for the accuracy achievable within the time limit.\n",
+    "- `estimate_time_only` - This flag enables users to obtain an estimate for the QPU time required to execute the circuit with QESEM. If it is set to `\"analytical\"`, an upper bound of the QPU time is calculated without consuming any QPU usage. This estimation has a 30-minute resolution (e.g., 30 minutes, 60 minutes, 90 minutes, etc.). It is typically pessimistic, and can only be obtained for single Pauli observables or sums of Paulis without intersecting supports (e.g. Z0+Z1). It is primarily useful for comparing the complexity levels of different parameters provided by the user (circuit, accuracy, etc.).\n",
+    "To obtain a more accurate QPU time estimation, set this flag to `\"empirical\"`. Although this option requires running a small number of circuits, it provides a much tighter QPU time estimation. This estimation has a 5-minute resolution (e.g., 20 minutes, 25 minutes, 30 minutes, etc.).\n",
```

Comment:
I'm guessing this is a `Batch` workload to perform an `empirical` estimation? Is there a rough idea of how much the estimate might cost a user? Is this doing noise sampling or more?